pom.xml

```diff
@@ -479,6 +480,22 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-invoker-plugin</artifactId>
           <version>3.1.0</version>
+          <configuration>
+            <debug>${it.debug}</debug>
+            <javaHome>${java.home}</javaHome>
+            <environmentVariables>
+              <JAVA_HOME>${java.home}</JAVA_HOME>
+            </environmentVariables>
+            <projectsDirectory>src/it</projectsDirectory>
+            <timeoutInSeconds>300</timeoutInSeconds>
```

Comment:
What does the timeoutInSeconds apply to? I notice that when the jetty maven plugin integration tests run, they often seem to pause a long time - is that anything to do with this timeout?

Reply:
max execution time. pause? maybe you mean a bit longer to execute than a junit test?